conda_index/index/__init__.py

```diff
@@ -557,7 +439,7 @@ def index(
             # begin non-stop "extract packages into cache";
             # extract_subdir_to_cache manages subprocesses. Keeps cores busy
             # during write/patch/update channeldata steps.
-            def extract_subdirs_to_cache():
+            def extract_subdirs_to_cache():  # is the 'prepare' step in 'index_prepared_subdir'
```

Comment:
`index_prepared_subdir` was renamed, we could say that we have to load metadata into the cache before we can generate-repodata-out-of-it.